internal/gamepad/gamepad_linux.go

```diff
@@ -200,15 +206,15 @@ func (*nativeGamepadsImpl) openGamepad(gamepads *gamepads, path string) (err err
 		buttonCount++
 	}
 	for code := 0; code < _ABS_CNT; code++ {
-		n.absMap[code] = -1
 		if !isBitSet(absBits, code) {
 			continue
 		}
 		if code >= _ABS_HAT0X && code <= _ABS_HAT3Y {
 			n.absMap[code] = hatCount
-			hatCount++
 			// Skip Y.
 			code++
+			n.absMap[code] = hatCount
+			hatCount++
```

Comment:
Is there an intention for this change?

Reply:
Yes - the lookup may now also happen on the Y coordinate.

But I noticed that I forgot to update the comment. Fixing.